Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -31,6 +33,7 @@ import controlPanel from './controlPanel.html';
 
 // Force the loading of HttpDataAccessHelper to support gzip decompression
 import 'vtk.js/Sources/IO/Core/DataAccessHelper/HttpDataAccessHelper';
+import { BlendMode } from '../../../../Rendering/Core/VolumeMapper/Constants';
```

Comment:
i think this should be with 'vtk.js/Sources/Rendering/Core/VolumeMapper/Constants'